src/main/java/uk/gov/justice/probation/courtcaseservice/jpa/entity/CourtCaseEntity.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.probation.courtcaseservice.jpa.entity;
+
+import com.vladmihalcea.hibernate.type.json.JsonBinaryType;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Table(name = "COURT_CASE")
+@TypeDef(name = "jsonb", typeClass = JsonBinaryType.class)
+public class CourtCaseEntity implements Serializable {
+
+    @Id
+    @Column(name = "ID")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "CASE_ID")
+    private Long caseId;
+
+    @Column(name = "CASE_NO")
+    private String caseNo;
+
+    @Column(name = "COURT_ID")
+    private Long courtId;
+
+    @Column(name = "COURT_ROOM")
+    private String courtRoom;
+
+    @Column(name = "SESSION_START_DATE")
+    private LocalDateTime sessionStartDate;
+
+    @Type(type = "jsonb")
+    @Column(columnDefinition = "jsonb", name = "DATA")
+    private CourtCasePropertiesEntity data;
```

Comment:
This is a good way of using JSONB but the downside is you're restricted to a specific schema for the JSONB field. If you want more flexibility you may want to have a normal JSONB field and manually parse the values to/from a String.